test/integration/suites/node-re-attestation/04-check-re-attest

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+source ./common
+
+docker-compose restart "spire-agent-a" "spire-agent-b" || fail-now "failed to stop services."
+
+# spire-agent-b can't re-attest because join_token implements trust on first use model.
+check-attested-agents 1
```

Comment:
can you verify why one of the agent fails, and the one that is failing is really the one with join token?

Reply:
761d9ec16eb6929f5ea21d2fdf0c523c6acb9bec